### PR TITLE
Fixes inconsistent monkey recycler description.

### LINKED ISSUE #10565: Monkey Recycler description inconsistent

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/-tg-station/issues/10565.

babby's first actual bug fix. changed plaintext to constant so the monkey recycler automatically updates with the number of monkeys required. this was correct later on in the code so I assume it's just an oversight

### CODE REVIEW

**Comment by phil235** [comment]
You can't do it like that. You have to update the desc var in a proc.

**Comment by Ikarrus** [comment]
Make sure you try compiling your code before you submit it in the future. You can catch these problems and fix them much earlier and easier than waiting for travis to tell you.

File: code/modules/food&drinks/kitchen machinery/monkeyrecycler.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/monkey_recycler
 	name = "monkey recycler"
-	desc = "A machine used for recycling dead monkeys into monkey cubes. It requires 5 monkeys per cube."
+	desc = "A machine used for recycling dead monkeys into monkey cubes. It requires [required_grind] monkeys per cube."
```
Comment (xxalpha) [inline code/modules/food&drinks/kitchen machinery/monkeyrecycler.dm]: If you compile you will know you can't do this: you can't set default values with non constants. Including a var in the default value will not compile.

**Comment by bgobandit** [comment]
:honkman:

tested it, it both compiles and works now

**Pull request closed without merging**